site/pages/components/link.tsx

```diff
@@ -0,0 +1,643 @@
+              type: 'string',
+              default: 'link',
+              description: (
+                <>
+                  Allows users to add event originator custom name e.g.
+                  &apos;newskit-inline-link&apos;.
+                </>
+              ),
+            },
+            {
+              name: 'external',
+              type: 'boolean',
+              description: (
+                <>
+                  If true, renders the &apos;external&apos; icon next to the
+                  LinkInline body content.
+                  <br />
+                  <br />
+                  Note - when a LinkInline renders, it automatically checks if
+                  the passed href is external or internal to the website where
+                  the link is used. If the href is external, an icon indicating
+                  this will be rendered after (trailing) the label
+                </>
+              ),
+            },
+          ],
+          overridesRows: [
+            {
+              attribute: 'stylePreset',
+              type: 'MQ<string>',
+              default: 'linkInline',
+              description:
+                'If provided, overrides the stylePresets of the LinkInline.',
+            },
+            {
+              attribute: 'transitionPreset',
+              type: 'MQ<string>',
+              default: ['fontColorChange', 'iconColorChange'],
+              description:
+                'If provided, overrides the transitionPresets of the LinkInline.',
+            },
+            {
+              attribute: 'spaceInline',
+              type: 'MQ<string>',
+              default: 'space010',
+              description:
+                'If provided overrides the gap between each element in the LinkInline. e.g. icon and text.',
+            },
+            {
+              attribute: 'externalIcon.size',
+              type: 'MQ<string>',
+              default: 'iconSize010',
+              description: (
+                <>
+                  If provided, overrides the size of the external icon, that
+                  appears after (trailing) the label of the LinkInline.
+                  <br />
+                  <br />
+                  Note - it is also possible to set the icon size by passing it
+                  directly as a <InlineCode>size</InlineCode> prop to the icon.
+                  However, by doing this you will override the{' '}
+                  <InlineCode>iconSize</InlineCode> passed from overrides.
+                </>
+              ),
+            },
+            ...(commonLogicalProps() as OverridesRowsProps[]),
+          ],
+          overridesSummary:
+            'The LinkInline has a range of predefined elements and attributes that can be overridden to define its appearance.',
+          propsFooter: (
+            <>
+              <InlineMessage
+                icon={infoIcon}
+                role="region"
+                aria-label="props link info"
```

Comment:
You have an accessibility error, I think it comes from that InlineMessage and the fact you have 2 messages with the same aria-label.
```suggestion
                aria-label="props LinkInline info"
```